### PR TITLE
Added github templates.

### LINKED ISSUE #9: PR template missing

### PR DESCRIPTION
**Pull request by alexemdesgagne**

### CODE REVIEW

**Review by supertoto29 — COMMENTED** [review]
Wouldn't be possible to remove the priority of the types for the templates? This would reduce the amount of files.

**Comment by alexemdesgagne** [comment]
> Wouldn't be possible to remove the priority of the types for the templates? This would reduce the amount of files.

What do you think @Kam0ss ?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+blank_issues_enabled: false
+# TODO: enter appropriate contacts
```
Comment (Kam0ss) [inline .github/ISSUE_TEMPLATE/config.yml]: Don't forget to change the contacts

**Review by Kam0ss — DISMISSED** [review]
LGTM, don't forget to change the contacts

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (alexemdesgagne) [reply .github/ISSUE_TEMPLATE/config.yml]: Done!

**Review by supertoto29 — APPROVED** [review]
LGTM

**Pull request merged**